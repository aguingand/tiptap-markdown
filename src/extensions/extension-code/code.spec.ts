import { expect, test } from "vitest";
import { parse, serialize } from "../../../__tests__/utils";


test('parse markdown', () => {
    expect(parse('`example`')).toMatchSnapshot();
});
test('parse html', () => {
    expect(parse('<code>example</code>')).toMatchSnapshot();
});
test.skip('serialize', () => {
    expect(serialize('<code>example</code>')).toEqual('`example`');
});
