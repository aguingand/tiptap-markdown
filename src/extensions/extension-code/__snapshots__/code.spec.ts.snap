// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`parse html 1`] = `
[
  {
    "content": [
      {
        "marks": [
          {
            "type": "code",
          },
        ],
        "text": "example",
        "type": "text",
      },
    ],
    "type": "paragraph",
  },
]
`;

exports[`parse markdown 1`] = `
[
  {
    "content": [
      {
        "marks": [
          {
            "type": "code",
          },
        ],
        "text": "example",
        "type": "text",
      },
    ],
    "type": "paragraph",
  },
]
`;
